### DISCUSSION TITLE
New collection review: cisco.dnac

### DISCUSSION THREAD

**Discussion by wastorga** [ansible-collections/ansible-inclusion#45] [Invalid]
Hello, I'd like to submit the cisco.dnac collection for review:

Collection provides functionality to manage and automate your Cisco DNA Center environment with Ansible.

Ansible Galaxy: https://galaxy.ansible.com/cisco/dnac
Github: https://github.com/cisco-en-programmability/dnacenter-ansible
Issues Tracker:  https://github.com/cisco-en-programmability/dnacenter-ansible/issues
Github handles: @wastorga @racampos @jbogarin @fmunozmiranda @bvargasre
Is the collection part of Automation Hub: Yes
We meet [Collection Requirements](https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst): Yes

**Comment by MarkusTeufelberger** [comment]
Duplicate https://github.com/ansible-collections/ansible-inclusion/discussions/20

**Reply by felixfontein** [reply]
I created a new category "invalid" for this and moved the application there.
